showroom/styles.js

```diff
@@ -42,3 +42,6 @@ require('../src/collapsable-section/examples/collapsableSection.scss');
 require('../src/background-dimmer/examples/backgroundDimmer.scss');
 require('../src/background-dimmer/examples/cover.png');
 require('../src/background-dimmer/examples/rating.png');
+
+require('../src/nav-bar/examples/navbar.scss');
```

Comment:
![image](https://cloud.githubusercontent.com/assets/4029499/18254955/c567fc90-73a3-11e6-9865-782cc9ca70a5.png)
typo: it should be `navBar.scss` in camelCase (thank you CI!)